test/api.test.ts

```diff
@@ -0,0 +1,31 @@
+import { describe, it, expect } from 'vitest'
```

Comment:
I would personally use [asserton lib](https://nodejs.org/api/assert.html). Usecase is super simple for us than having a full test runner :D